spec/factories/claim/base_claims.rb

```diff
@@ -14,9 +14,11 @@
     providers_ref       { random_providers_ref }
     disable_for_state_transition nil
     after(:create) do |claim|
-      defendant = create(:defendant, claim: claim)
-      create(:representation_order, defendant: defendant, representation_order_date: 380.days.ago)
-      claim.reload
+      unless claim.defendants.present?
```

Comment:
This is a big 🅱️ 👣  for me! Allows us to define a *claim* factory providing the defendants directly, rather than taking the existent convoluted approach of `after_create` having to clear any existent defendants autogenerated by the factory and then assigning the actual ones we want to have.